tests/flow/test_pending_queries_limit.py

```diff
@@ -8,19 +9,43 @@
 # 3. test overflowing the server when there's no limit
 #    expect not to get any exceptions
 
+con = None
 error_encountered = False
 
 GRAPH_NAME = "max_pending_queries"
+# note: if test_03_overflow_with_limit is flaky 
+# on machines with many cores on some runs the rate at which the queries been executed by the workers 
+# might be greater than the insertion to the work queues. 
+# This ends up were there is at most one job in the queue which failed
+# test_03_overflow_with_limit. Increasing the execution time of the query should resolved this.
 SLOW_QUERY = "UNWIND range (0, 100000) AS x WITH x WHERE (x / 2) = 50  RETURN x"
 
-def issue_query(conn, q):
-    global error_encountered
-
+def issue_query(q):
+    global con
     try:
-        conn.execute_command("GRAPH.QUERY", GRAPH_NAME, q)
+        con.execute_command("GRAPH.QUERY", GRAPH_NAME, q)
+        return False
     except Exception as e:
         assert "Max pending queries exceeded" in str(e)
-        error_encountered = True
+        return True
+
+def run_test_multiproc(env, n_procs, fn, args=tuple()):
```

Comment:
Why do we need to set  `args` default value to `tuple()` ?

Reply:
So that args can be omitted from the calling statement.
I wanted this function to have a general API so it can be ported in the future.